packages/maps/src/lib/mapControlGroup/MapControlGroup.stories.svelte

```diff
@@ -22,78 +40,97 @@
 	parameters={{
 		layout: 'fullscreen'
 	}}
+	argTypes={{
+		x: newSelectedStringArg(['left', 'center', 'right'], 'left'),
+		y: newSelectedStringArg(['top', 'center', 'bottom'], 'top'),
+		xOffset: newSelectedStringArg(['md', 'lg'], 'md'),
+		yOffset: newSelectedStringArg(['md', 'lg'], 'md')
+	}}
 />
 
 <Template let:args>
-	<MapControlGroup {...args} />
+	<div class="w-[100dvw] h-[100dvh] bg-color-bg-primary relative">
+		<MapControlGroup {...args}>
+			<Button variant="square" emphasis="secondary" class="pointer-events-auto">🗺️</Button>
+		</MapControlGroup>
+	</div>
 </Template>
 
-<Story name="Positioning labels">
-	<MapApp>
-		<Map
-			options={{
-				style: os_light_vts,
-				transformRequest: appendOSKeyToUrl(OS_KEY)
-			}}
-		>
-			{#each Object.keys(MapControlGroupPositions) as position}
-				<MapControlGroup {position}>
-					<p class="bg-color-container-level-1 text-color-text-primary p-2">
-						{position}
+<Story name="Default" source />
+
+<Story name="Positions">
+	<div class="w-[100dvw] h-[100dvh] bg-color-bg-primary relative">
+		{#each yPositions as yPos (yPos)}
+			{#each xPositions as xPos (xPos)}
+				<MapControlGroup y={yPos} x={xPos}>
+					<p class="bg-color-container-level-1 text-color-text-primary p-2 capitalize">
+						{yPos}
+						{xPos}
```

Comment:
Would be clearer to display as something like `x={xPos} y={yPos}` or `y={yPos} x={xPos}`